lib/leanplum_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module LeanplumApi
-  VERSION = '4.0.3'
```

Comment:
This is actually a minor bump (not a patch), given that it's new, backward-compatible feature, per [semver](https://semver.org/).

Reply:
ok, sure, in contrast to the prev change that was a bug fix this is adding a new feature, so yeah, will change!

Reply:
@slpsys it's changed now

Reply:
👍